### PR TITLE
[Warp Client] Refined modes explanation

### PR DESCRIPTION
**Pull request by marciocloudflare**
[PCX-3911](https://jira.cfops.it/browse/PCX-3911)

### CODE REVIEW

File: content/warp-client/_index.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ meta:
 
 # Cloudflare WARP client
 
-The Cloudflare WARP client allows individuals and organizations to have a faster, more secure, and more private experience online. The WARP client has several modes, to better suit different connection needs. To learn more about WARP and the several modes, refer to [WARP modes](/warp-client/warp-modes/).
+The Cloudflare WARP client allows individuals and organizations to have a faster, more secure, and more private experience online. The WARP client sits between your device and the Internet, and has [several connection modes](/warp-client/warp-modes/) to better suit different needs.
 
-For more information on how to use WARP to enhance your Zero Trust experience, refer to the [Cloudflare Zero Trust documentation](/cloudflare-one/connections/connect-devices/warp/).
\ No newline at end of file
+Refer to [Get Started](/warp-client/get-started/) to learn more about which WARP version your should download for your operating system, including [iOS](https://apps.apple.com/us/app/id1423538627) and [Android](https://play.google.com/store/apps/details?id=com.cloudflare.onedotonedotonedotone).
```
Comment (ranbel) [inline content/warp-client/_index.md]: Why do we specifically call out iOS and Android?

File: content/warp-client/_index.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ meta:
...
+The Cloudflare WARP client allows individuals and organizations to have a faster, more secure, and more private experience online. The WARP client sits between your device and the Internet, and has [several connection modes](/warp-client/warp-modes/) to better suit different needs.
 
-For more information on how to use WARP to enhance your Zero Trust experience, refer to the [Cloudflare Zero Trust documentation](/cloudflare-one/connections/connect-devices/warp/).
\ No newline at end of file
+Refer to [Get Started](/warp-client/get-started/) to learn more about which WARP version your should download for your operating system, including [iOS](https://apps.apple.com/us/app/id1423538627) and [Android](https://play.google.com/store/apps/details?id=com.cloudflare.onedotonedotonedotone). 
+
+If you are looking for the enterprise version of WARP, refer to the [Cloudflare Zero Trust documentation](/cloudflare-one/connections/connect-devices/warp/).
```
Comment (ranbel) [inline content/warp-client/_index.md:15]: Should we specify that this WARP client is "consumer WARP"?

**Review by ranbel — APPROVED** [review]

File: content/warp-client/_index.md (hunk 1 above)
Comment (marciocloudflare) [reply content/warp-client/_index.md]: Good question. I guess I spent so much time going around with how to call out that the mobile apps have a totally different name, I missed the fact that like it is right now I don't even mention the names lol

File: content/warp-client/_index.md (hunk 2 above)
Comment (marciocloudflare) [reply content/warp-client/_index.md:15]: Not sure. There's no "official" name for the consumer side of WARP, despite the WARP Client naming we have in the tiles.

**Pull request merged**